taskcluster/scripts/toolchain/android-gradle-dependencies.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -ex
+
+function get_abs_path {
+    local file_path="$1"
+    echo "$( cd "$(dirname "$file_path")" >/dev/null 2>&1 ; pwd -P )"
+}
+
+CURRENT_DIR="$(get_abs_path $0)"
+PROJECT_DIR="$(get_abs_path $CURRENT_DIR/../../../..)"
+
+pushd $PROJECT_DIR
+
+. taskcluster/scripts/toolchain/android-gradle-dependencies/before.sh
+
+# We build everything to be sure to fetch all dependencies
+./gradlew --no-daemon -PgoogleRepo='http://localhost:8081/nexus/content/repositories/google/' -PjcenterRepo='http://localhost:8081/nexus/content/repositories/jcenter/' assemble assembleAndroidTest bundle test lint ktlint detekt
```

Comment:
I wanted to factorize all these steps into one (namely `build`) which technically triggers `test`, `lint` and `assemble` https://docs.gradle.org/current/userguide/java_plugin.html#lifecycle_tasks, but it turned out it doesn't build `bundle` nor `ktlint`, nor some variants. So I think we have the less worst list, as far as I can tell. I'll have to communicate about this list.

I also wanted to `--dry-run` in order to save compilation time, but it doesn't fetch all dependencies. See https://firefox-ci-tc.services.mozilla.com/tasks/Gb1PceneSEm9doboiEu3AQ/runs/0/logs/https%3A%2F%2Ffirefox-ci-tc.services.mozilla.com%2Fapi%2Fqueue%2Fv1%2Ftask%2FGb1PceneSEm9doboiEu3AQ%2Fruns%2F0%2Fartifacts%2Fpublic%2Flogs%2Flive.log#L266